index.bs

```diff
@@ -138,6 +160,108 @@ call stack, at the time of the crash, to be included in a crash report.
 <xmp class="http">Document-Policy: include-js-call-stacks-in-crash-reports</xmp>
 </div>
 
+Processing Model {#processing}
+================
+
+Note: Browser architectures vary, and is it possible that a browser exists which
+cannot support this specification as written due to its architecture.
+
+This processing model makes certain assumptions about the underlying process
+architecture. Namely:
+
+* Each [=Document=] is controlled by a single process, such that if that process
+   crashes, the controlled [=Document=] will immediately and abruptly terminate.
+
+   Note: It is not required that all such [=Documents=] are related to each
+   other in any way other than being controlled by the same process.
+
+* A separate <dfn>coordinator process</dfn> exists which is responsible for
+   monitoring the processes which control documents, and which is expected to
+   survive even when those processes crash. This process is able to observe the
+   document process crash.
+
+* The coordinator process maintains enough information about the state of each
+   [=Document=] that after a process crash, it can reconstruct the frame tree
```

Comment:
Not sure I have concrete feedback, but "frame tree" feels out of place here. Cursory search shows that Gecko and WebKit both use the same concept and phrasing (Ladybird seems to have the concept without the exact wording), so probably not a problem.